src/cmstats.c

```diff
@@ -49,6 +50,25 @@ s_duplicator (const void *self)
     return (void*) fty_proto_dup ((fty_proto_t*) self);
 }
 
+static double
+safe_atof (const char *str)
+{
+    char *endptr;
+    errno = 0;
+    double ret = strtod (str, &endptr);
+    // we could not process whole str
+    if (ret == 0.0 && endptr != NULL && endptr[0] != '\0') {
+        log_error ("strtod received %s with dangling non-number argument '%s'", str, endptr);
+        //ret = "NAN";
+    }
+    // overflow or underflow
+    if (errno == ERANGE) {
```

Comment:
Should we complain about actually any non-zero `errno`? Should `strtod()` reset it to okay state if all went well?